argilla-server/src/argilla_server/_app.py

```diff
@@ -47,6 +47,7 @@
 @contextlib.asynccontextmanager
 async def app_lifespan(app: FastAPI):
     # See https://fastapi.tiangolo.com/advanced/events/#lifespan
+    check_telemetry()
```

Comment:
I think we should rename the function name to show better what's doing:

```suggestion
    show_telemetry_warning()
```